libexec/python/docker/api.py

```diff
@@ -51,13 +49,34 @@ def create_runscript(cmd,base_dir):
     output_file = write_file(runscript,content)
     return output_file
 
-def create_envfile(env, base_dir):
-    envfile = "%s/environment" % base_dir
+
+def create_envfile(base_dir,env=None,manifest=None):
+    '''create_envfile will generate a file called environment
+    in the base_dir of an image
+    :param base_dir: the base_dir to write the file to
+    :param manifest: the manifest to generate the Env for.
+    :param env: a list of environment variables to export, if manifest
+    is not provided
+    '''
+    if env == None:
+        if manifest == None:
+            logger.error("env or manifest must be provided, exiting")
+            sys.exit(1)    
+        env = get_config(manifest, 'Env').split("\n")
```

Comment:
It's because as a developer, I don't find it intuitive to get "env" via get_config(spec="env") and then have to split that by newline before giving to the function to generate the environment. I do, however, find it intuitive to grab a manifest, and then generate an environment from it.